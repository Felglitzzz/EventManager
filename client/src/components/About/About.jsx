import React from 'react';
/**
 * About Class
 */
export default class About extends React.Component {
  /**
   * Nav
   * @returns {react} About components
   */
  render() {
    return (
      <div className="container-fluid">
        <div className="my-5 mx-5 no-margin-hr">
          <p className="display-3 text-center text-orange homefont">What we do</p>
          <p className="text-center lead testfont">
            We are the leading event technology platform in Africa. We cover your event and make it
            more
            <span className="text-orange">
              {' '}
              <u>colourful</u>.{' '}
            </span>
            Our technology is a foremost and deft custodian for your events.
          </p>
        </div>
      </div>
    );
  }
}
