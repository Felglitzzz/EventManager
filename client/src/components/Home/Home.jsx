import React from 'react';
/**
 * Home Class
 */
export default class Home extends React.Component {
  /**
   * componentDidMount
   * @returns { object } isAuthenticated
   */
  componentWillMount() {
    this.setState({ isAuthenticated: false });
  }
  /**
   * Nav
   * @returns {Home} Home components
   */
  render() {
    return (
      <div>
        <div className="container-fluid half_screen img-fluid d-flex align-items-center">
          <div
            className="container-fluid carousel slide"
            data-ride="carousel"
            id="carouselExampleSlidesOnly"
          >
            <div className="carousel-inner h-auto">
              <div className="carousel-item active">
                <p className="text-center text-light homefont">
                  We help your event communicate beauty
                </p>
              </div>
              <div className="carousel-item">
                <p className="text-center text-light homefont">
                  Your events, Amplified! Managing It, Simplified!
                </p>
              </div>
            </div>
            <div className="d-flex justify-content-center">
              <a
                className="d-flex marg btn border-white btn-outline-orange aconform text-white px-5"
                href="#discover"
                type="button"
              >
                Explore
              </a>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
