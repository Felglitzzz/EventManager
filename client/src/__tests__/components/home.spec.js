import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../components/Home/Home';
import About from '../../components/About/About';

describe('Home Page', () => {
  it('should render the home page correctly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('About Page', () => {
  it('should render the home page correctly', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
