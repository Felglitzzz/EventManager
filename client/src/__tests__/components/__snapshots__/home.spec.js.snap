// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should render the home page correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="my-5 mx-5 no-margin-hr"
      >
        <p
          className="display-3 text-center text-orange homefont"
        >
          What we do
        </p>
        <p
          className="text-center lead testfont"
        >
          We are the leading event technology platform in Africa. We cover your event and make it more
          <span
            className="text-orange"
          >
             
            <u>
              colourful
            </u>
            .
             
          </span>
          Our technology is a foremost and deft custodian for your events.
        </p>
      </div>,
      "className": "container-fluid",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="display-3 text-center text-orange homefont"
          >
            What we do
          </p>,
          <p
            className="text-center lead testfont"
          >
            We are the leading event technology platform in Africa. We cover your event and make it more
            <span
              className="text-orange"
            >
               
              <u>
                colourful
              </u>
              .
               
            </span>
            Our technology is a foremost and deft custodian for your events.
          </p>,
        ],
        "className": "my-5 mx-5 no-margin-hr",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "What we do",
            "className": "display-3 text-center text-orange homefont",
          },
          "ref": null,
          "rendered": "What we do",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "We are the leading event technology platform in Africa. We cover your event and make it more",
              <span
                className="text-orange"
              >
                 
                <u>
                  colourful
                </u>
                .
                 
              </span>,
              "Our technology is a foremost and deft custodian for your events.",
            ],
            "className": "text-center lead testfont",
          },
          "ref": null,
          "rendered": Array [
            "We are the leading event technology platform in Africa. We cover your event and make it more",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  <u>
                    colourful
                  </u>,
                  ".",
                  " ",
                ],
                "className": "text-orange",
              },
              "ref": null,
              "rendered": Array [
                " ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "colourful",
                  },
                  "ref": null,
                  "rendered": "colourful",
                  "type": "u",
                },
                ".",
                " ",
              ],
              "type": "span",
            },
            "Our technology is a foremost and deft custodian for your events.",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="my-5 mx-5 no-margin-hr"
        >
          <p
            className="display-3 text-center text-orange homefont"
          >
            What we do
          </p>
          <p
            className="text-center lead testfont"
          >
            We are the leading event technology platform in Africa. We cover your event and make it more
            <span
              className="text-orange"
            >
               
              <u>
                colourful
              </u>
              .
               
            </span>
            Our technology is a foremost and deft custodian for your events.
          </p>
        </div>,
        "className": "container-fluid",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="display-3 text-center text-orange homefont"
            >
              What we do
            </p>,
            <p
              className="text-center lead testfont"
            >
              We are the leading event technology platform in Africa. We cover your event and make it more
              <span
                className="text-orange"
              >
                 
                <u>
                  colourful
                </u>
                .
                 
              </span>
              Our technology is a foremost and deft custodian for your events.
            </p>,
          ],
          "className": "my-5 mx-5 no-margin-hr",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "What we do",
              "className": "display-3 text-center text-orange homefont",
            },
            "ref": null,
            "rendered": "What we do",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "We are the leading event technology platform in Africa. We cover your event and make it more",
                <span
                  className="text-orange"
                >
                   
                  <u>
                    colourful
                  </u>
                  .
                   
                </span>,
                "Our technology is a foremost and deft custodian for your events.",
              ],
              "className": "text-center lead testfont",
            },
            "ref": null,
            "rendered": Array [
              "We are the leading event technology platform in Africa. We cover your event and make it more",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    <u>
                      colourful
                    </u>,
                    ".",
                    " ",
                  ],
                  "className": "text-orange",
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "colourful",
                    },
                    "ref": null,
                    "rendered": "colourful",
                    "type": "u",
                  },
                  ".",
                  " ",
                ],
                "type": "span",
              },
              "Our technology is a foremost and deft custodian for your events.",
            ],
            "type": "p",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Home Page should render the home page correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container-fluid half_screen img-fluid d-flex align-items-center"
      >
        <div
          className="container-fluid carousel slide"
          data-ride="carousel"
          id="carouselExampleSlidesOnly"
        >
          <div
            className="carousel-inner h-auto"
          >
            <div
              className="carousel-item active"
            >
              <p
                className="text-center text-light homefont"
              >
                We help your event communicate beauty
              </p>
            </div>
            <div
              className="carousel-item"
            >
              <p
                className="text-center text-light homefont"
              >
                Your events, Amplified! Managing It, Simplified!
              </p>
            </div>
          </div>
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container-fluid carousel slide"
          data-ride="carousel"
          id="carouselExampleSlidesOnly"
        >
          <div
            className="carousel-inner h-auto"
          >
            <div
              className="carousel-item active"
            >
              <p
                className="text-center text-light homefont"
              >
                We help your event communicate beauty
              </p>
            </div>
            <div
              className="carousel-item"
            >
              <p
                className="text-center text-light homefont"
              >
                Your events, Amplified! Managing It, Simplified!
              </p>
            </div>
          </div>
        </div>,
        "className": "container-fluid half_screen img-fluid d-flex align-items-center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="carousel-inner h-auto"
          >
            <div
              className="carousel-item active"
            >
              <p
                className="text-center text-light homefont"
              >
                We help your event communicate beauty
              </p>
            </div>
            <div
              className="carousel-item"
            >
              <p
                className="text-center text-light homefont"
              >
                Your events, Amplified! Managing It, Simplified!
              </p>
            </div>
          </div>,
          "className": "container-fluid carousel slide",
          "data-ride": "carousel",
          "id": "carouselExampleSlidesOnly",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="carousel-item active"
              >
                <p
                  className="text-center text-light homefont"
                >
                  We help your event communicate beauty
                </p>
              </div>,
              <div
                className="carousel-item"
              >
                <p
                  className="text-center text-light homefont"
                >
                  Your events, Amplified! Managing It, Simplified!
                </p>
              </div>,
            ],
            "className": "carousel-inner h-auto",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="text-center text-light homefont"
                >
                  We help your event communicate beauty
                </p>,
                "className": "carousel-item active",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "We help your event communicate beauty",
                  "className": "text-center text-light homefont",
                },
                "ref": null,
                "rendered": "We help your event communicate beauty",
                "type": "p",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="text-center text-light homefont"
                >
                  Your events, Amplified! Managing It, Simplified!
                </p>,
                "className": "carousel-item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Your events, Amplified! Managing It, Simplified!",
                  "className": "text-center text-light homefont",
                },
                "ref": null,
                "rendered": "Your events, Amplified! Managing It, Simplified!",
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container-fluid half_screen img-fluid d-flex align-items-center"
        >
          <div
            className="container-fluid carousel slide"
            data-ride="carousel"
            id="carouselExampleSlidesOnly"
          >
            <div
              className="carousel-inner h-auto"
            >
              <div
                className="carousel-item active"
              >
                <p
                  className="text-center text-light homefont"
                >
                  We help your event communicate beauty
                </p>
              </div>
              <div
                className="carousel-item"
              >
                <p
                  className="text-center text-light homefont"
                >
                  Your events, Amplified! Managing It, Simplified!
                </p>
              </div>
            </div>
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container-fluid carousel slide"
            data-ride="carousel"
            id="carouselExampleSlidesOnly"
          >
            <div
              className="carousel-inner h-auto"
            >
              <div
                className="carousel-item active"
              >
                <p
                  className="text-center text-light homefont"
                >
                  We help your event communicate beauty
                </p>
              </div>
              <div
                className="carousel-item"
              >
                <p
                  className="text-center text-light homefont"
                >
                  Your events, Amplified! Managing It, Simplified!
                </p>
              </div>
            </div>
          </div>,
          "className": "container-fluid half_screen img-fluid d-flex align-items-center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="carousel-inner h-auto"
            >
              <div
                className="carousel-item active"
              >
                <p
                  className="text-center text-light homefont"
                >
                  We help your event communicate beauty
                </p>
              </div>
              <div
                className="carousel-item"
              >
                <p
                  className="text-center text-light homefont"
                >
                  Your events, Amplified! Managing It, Simplified!
                </p>
              </div>
            </div>,
            "className": "container-fluid carousel slide",
            "data-ride": "carousel",
            "id": "carouselExampleSlidesOnly",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="carousel-item active"
                >
                  <p
                    className="text-center text-light homefont"
                  >
                    We help your event communicate beauty
                  </p>
                </div>,
                <div
                  className="carousel-item"
                >
                  <p
                    className="text-center text-light homefont"
                  >
                    Your events, Amplified! Managing It, Simplified!
                  </p>
                </div>,
              ],
              "className": "carousel-inner h-auto",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p
                    className="text-center text-light homefont"
                  >
                    We help your event communicate beauty
                  </p>,
                  "className": "carousel-item active",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "We help your event communicate beauty",
                    "className": "text-center text-light homefont",
                  },
                  "ref": null,
                  "rendered": "We help your event communicate beauty",
                  "type": "p",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <p
                    className="text-center text-light homefont"
                  >
                    Your events, Amplified! Managing It, Simplified!
                  </p>,
                  "className": "carousel-item",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Your events, Amplified! Managing It, Simplified!",
                    "className": "text-center text-light homefont",
                  },
                  "ref": null,
                  "rendered": "Your events, Amplified! Managing It, Simplified!",
                  "type": "p",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
